Utilities/MCStepLogger/include/MCStepLogger/BasicMCAnalysis.h

```diff
@@ -0,0 +1,108 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+//****************************************************************************
+//* This file is free software: you can redistribute it and/or modify        *
+//* it under the terms of the GNU General Public License as published by     *
+//* the Free Software Foundation, either version 3 of the License, or        *
+//* (at your option) any later version.                                      *
+//*                                                                          *
+//* Primary Authors: Benedikt Volkel <benedikt.volkel@cern.ch>               *
+//*                                                                          *
+//* The authors make no claims about the suitability of this software for    *
+//* any purpose. It is provided "as is" without express or implied warranty. *
+//****************************************************************************
+
+#ifndef BASIC_MCANALYSIS_H_
+#define BASIC_MCANALYSIS_H_
+
+#include <unordered_map>
+
+#include "MCStepLogger/MCAnalysis.h"
+
+namespace o2
+{
+namespace mcstepanalysis
+{
+
+class BasicMCAnalysis : public MCAnalysis
```

Comment:
could you please add a short docu for what this class is doing/providing?

Reply:
on it